aws/services/postgres/main.tf

```
@@ -0,0 +1,30 @@
+resource "random_string" "db-password" {
+  length  = 32
+  upper   = true
+  number  = true
+  special = false
+}
+resource "aws_security_group" "db_security_group" {
+  vpc_id      = local.vpc
+  name        = "db-security-group"
+  description = "Allow all inbound for Postgres"
+ingress {
+    from_port   = 5432
+    to_port     = 5432
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+resource "aws_db_instance" "postgres_db_instance" {
+  identifier             = "postgres-db"
+  name                   = "postgres-db"
+  instance_class         = "db.t2.micro"
```

just noticed that `db.t4g.micro` is even cheaper 
```suggestion
  instance_class         = "db.t4g.micro"
```